app/controllers/locations_controller.rb

```diff
@@ -2,11 +2,7 @@ class LocationsController < ApplicationController
   def index
     expires_in Rails.application.config.cache_max_age, public: true
 
-    @locations = Location.current.includes(:address, :booking_location)
-
-    unless params[:include_hidden_locations]
-      @locations = @locations.where(hidden: false)
-    end
+    @locations = Location.externally_visible(include_hidden_locations: params[:include_hidden_locations])
```

Comment:
You could replace the explicit kwarg with just `Locations.externally_visible(params.slice(:include_hidden_locations))` as this would yield a hash that gets correctly parsed into kwargs. Alternatively you could hide the `params` access behind another private helper method such as `location_params` and call `Locations.externally_visible(location_params)` which trims this down a touch. 

You might argue the explicitness of passing the named kwarg, though 💭